<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrmViaCepComInterface</class>
 <widget class="QMainWindow" name="FrmViaCepComInterface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>406</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ViaCep Com Interface</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Consultar CEP: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lnEdtCEPBusca">
        <property name="inputMask">
         <string>99999-999</string>
        </property>
        <property name="text">
         <string>-</string>
        </property>
        <property name="maxLength">
         <number>9</number>
        </property>
        <property name="placeholderText">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pshBtnBuscar">
        <property name="text">
         <string>Buscar</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>35</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>CEP:                    </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lnEdtCEP"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Logradouro:     </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lnEdtLogradouro"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Complemento:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lnEdtComplemento"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Bairro:                </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lnEdtBairro"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Localidade:       </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lnEdtLocalidade"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>UF:                       </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lnEdtUF"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>IBGE:                   </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lnEdtIBGE"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_9">
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>GIA:                     </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lnEdtGIA"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_10">
      <item>
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Cont√©m Erro:   </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lnEdtErro"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
